### ISSUE TITLE
Make zotxt compatible with Zotero 5

### ISSUE THREAD

**Issue by okroeger** [egh/zotxt#1]
First, thanks a lot for your work! And, although I’m sure that you’ve got in your radar, please upgrade zotxt to work with Zotero 5. (I’m mostly posting this issue so that I’m notified once it’s done.) Thanks a lot!

**Comment by pierwill** [comment]
Hello! Love zotxt! Just a comment saying I'd love an update for Zotero 5!

**Comment by stasvlasov** [comment]
I am very grateful for your work as well! zotxt (with zotxt-emacs) makes it an enjoyable workflow in Emacs. I would very appreciate the update for Zotero 5. Thank you!

**Comment by okroeger** [comment]
I think @egh has moved development to https://gitlab.com/egh/zotxt/tree/zotero-5.0. I tried to use the add-on that he links to in the README.rst there, but it behaves weirdly.

**Comment by egh** [comment]
Hi @okroeger Honestly I go back and forth between gitlab and github: they should both work: I try to keep the code mirrored and monitor both.

I've finally finished up the move to Zotero 5 (that was quite a bit of work - apologies for the delay)

Please see my prerelease here:

https://github.com/egh/zotxt/releases/tag/v5.0.0rc

**Comment by andersjohansson** [comment]
Oh! This is the last missing piece for my migration to Zotero 5, since BBT (after what seems like heroic efforts) finally also works!
With my initial testing it seems to work just fine. The thing I missed, since I use org-zotxt, is the easyBib format. This was used for choosing items in `zotxt-choose-deferred` in zotxt.el.
Just pulling out the text-bibliography instead in `zotxt-choose-deferred` seemed to make it kind of work for now, but I guess this is slower with the whole CSL-machinery being invoked.

Do you think you will add this in some way again? I guess the easiest way would be to just include a translator like for easyKey, and that would also be fast I suppose. I could contribute it if needed!
Otherwise this translator could be distributed separately, or with zotxt.el

I don’t know if I’m the only one using org-zotxt, but I think it’s the best solution for keeping one’s writing in Emacs and org-mode while still using Zotero, and with some of the additions I have made I think it works very well! (when it doesn’t break ;-)). I think I’ve discussed some of these things earlier, but now I have put it all up here: https://gitlab.com/andersjohansson/emacs-zotxt-extra. Maybe some of it could be pulled into zotxt.el if you still want to maintain it. But all that are later questions.

For now, I think it’s time to celebrate this milestone!

**Comment by okroeger** [comment]
Thanks a lot, I had to migrate to Word for the time being (I only realised that I just broke my workflow after I had upgraded to Zotero 5), but will test it once the paper I’m writing is done.

**Comment by egh** [comment]
@andersjohansson Oh, I think `quickBib` was not in my test suite, so that was forgotten. I'll fix it. I use org-zotxt, but not as much as I'd like, so I am at least still maintaining it. :)

**Comment by egh** [comment]
@andersjohansson I have updated quickBib and also the paths response format, which allows viewing PDFs and other attachments in emacs. I've tested it out myself, and it seems to work. I've upload here: https://github.com/egh/zotxt/releases/edit/v5.0.0rc2

**Comment by andersjohansson** [comment]
Great, Everything works fine now! Thanks again!

**Comment by egh** [comment]
@andersjohansson My pleasure! Glad it's working again.

**Comment by egh** [comment]
Fixed a few more issues and finished the release: https://github.com/egh/zotxt/releases/tag/v5.0.0

**Comment by liamh** [comment]
I tried to load zotxt-5.0.0.xpi as shown on that release page into Zotero Standalone 5.0.30 for Linux, and got this message:

    The add-on "%S" could not be installed. It may be incompatible with this version of Zotero

and the result is that I do not have zotxt installed and cannot use in Zotero Standalone. Is there another way to do this that will work?

**Comment by egh** [comment]
@liamh Interesting. I am also on Zotero Standalone 5.0.30 for linux. I just re-installed zotxt 5.0.2 and it worked fine (https://github.com/egh/zotxt/releases/download/v5.0.2/zotxt-5.0.2.xpi) Can you try version 5.0.2? Thanks.

**Comment by liamh** [comment]
This works, thanks.

**Comment by egh** [comment]
Great! Thanks for the response.
